tools/apiview/parsers/apiview-treestyle-parser-schema/models.tsp

```diff
@@ -0,0 +1,150 @@
+model CodeFile {
+  /** The version of the parser */
+  VersionString: string;
+
+  /** Name the ocde file. Usually just the package name and version*/
+  Name: string;
+  
+  Language: string;
+  LanguageVariant: "Default" | "Spring" | "Android";
+  PackageName: string;
+  ServiceName: string;
+  PackageVersion: string;
```

Comment:
Are these actually required? If not they should be marked with ?

Reply:
@praveenkuttappan these are required right?

Reply:
PackageName and PackageVersion are required. @chidozieononiwu I don't think we use ServiceName anymore. Atleast not for all languages. So we should either remove it or mark it as optional.

Reply:
If we don't use it at all we should remove it.